gulpfile.babel.js

```diff
@@ -1,37 +1,32 @@
 'use strict';
 
-const path = require('path');
```

Comment:
As a result of [this](https://github.com/GoogleChrome/chromium-dashboard/pull/2380/files#r1003692372), had to switch all `require` to `import`.